spec/models/comment_spec.rb

```diff
@@ -1,5 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'adds comments' do
+    author = User.new(name: 'Tom', photo: 'profile_pic.jpeg', bio: 'Bio text template', posts_counter: 0)
+    post = Post.new(title: 'New test post', text: 'How are ', author: author, likes_counter: 0, comments_counter: 0)
+    post.save!
+
+    comment_creator = User.new(name: 'Jerry', photo: 'profile_pic.jpeg', bio: 'Bio template', posts_counter: 0)
+    comment_creator.save!
+
+    post.comments.create!(text: 'Hi World', author: comment_creator)
+    post.comments.create!(text: 'This is actually my second post', author: comment_creator)
+    expect(post.comments.length).to eql(2)
+  end
```

Comment:
- All the tests are not running successfully, please fix it.

![image](https://user-images.githubusercontent.com/55635977/174350252-be3758c2-2cda-4c9c-b113-d236efbf2d0f.png)

Reply:
Hi @Emmyn5600 Which branch are you running?

The code you shared in the snippet is not what I wrote. Check the ReadMe file as well on how to run test.

Reply:
Hi @Emmyn5600,

I have fixed the issues. 

Thanks a lot 👨🏽‍💻 />